components/View/index.jsx

```diff
@@ -131,19 +140,14 @@ class View extends Component {
     this.element.dispatchEvent(swipeEvent);
   };
 
-  /**
-   * Sets the navigator title when the component mounts.
-   */
-  routeWillEnter() {
-    this.props.setTitle(this.props.title);
-  }
-
   /**
    * Renders the HTML meta tags.
    * @returns {JSX}
    */
   renderMetaTags() {
-    const { meta, link, script, style } = this.props.head;
+    const {
```

Comment:
I think the linter was looking for:

```
const {
  meta,
  link,
  script,
  style,
} = this.props.head;
```

Reply:
There is a special option for the related rule, called `multiline` which is inactive.
https://eslint.org/docs/rules/object-curly-newline

So i think it's totally ok with it